bump.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/bash
+
+git config --global user.name "Kite Bot"
+git config --global user.email "kite-bot@heliostech.fr"
+git remote add authenticated-origin https://kite-bot:$GITHUB_API_KEY@github.com/${DRONE_REPO}
+git fetch authenticated-origin
+curl -sL https://deb.nodesource.com/setup_10.x | bash -
```

Comment:
we won't install nodejs in this context
the library will be present in the build environment